1-org/envs/shared/README.md

```diff
@@ -37,6 +37,20 @@ You can choose not to enable the Data Access logs by setting variable `data_acce
 1. Run `terraform plan` and review output.
 1. Run `terraform apply`
 
+### Run tf-wrapper.sh script locally
```

Comment:
Also I think there is a section right above this that also says `Run terraform locally`. We can replace that with this.

Reply:
done